catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/CategoryListFragment.java

```diff
@@ -617,8 +621,9 @@ private void initializeAdapter() {
 		} else {
 			resIds.addAll(OBJECT_LOOK);
 		}
-
-		return addHeader(toCategoryListItems(resIds), getString(R.string.formula_editor_object_general));
+		List<CategoryListItem> result = toCategoryListItems(resIds);
+		result.addAll(toCategoryListItems(OBJECT_COLOR_TOUCHES_COLOR, OBJECT_COLOR_TOUCHES_COLOR_PARAM));
```

Comment:
Just use `OBJECT_COLOR_COLLISION.subList(1,2)` and `OBJECT_COLOR_COLLISION.subList(1,2)` here to avoid making new variables and code duplication.